testsuite/integration/elytron/pom.xml

```diff
@@ -102,12 +102,26 @@
             <artifactId>jakarta-authentication</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>org.apache.directory.server</groupId>
             <artifactId>apacheds-test-framework</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kerby</groupId>
+            <artifactId>kerb-server-api-all</artifactId>
+            <version>2.0.3</version>
```

Comment:
Just a small thing, would be good to add a `version.org.apache.kerby`  property to the `properties` and reference that here and below.